CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
    * REMOVED: hard-coded tz alias map and associated logic [#5164](https://github.com/valhalla/valhalla/pull/5164)
    * REMOVED: `valhalla/filesystem` from the project in favor of the std equivalent [#5321](https://github.com/valhalla/valhalla/pull/5321)
 * **Bug Fix**
+   * FIXED: Add error handling to valhalla_build_tiles command in Docker [#5520](https://github.com/valhalla/valhalla/pull/5520)
```

Comment:
sorry, I should've said it and it seems I can't edit your fork: we keep an ascending changelog, new items go last

Reply:
fixed!